.github/workflows/invitation.yml

```
@@ -12,6 +12,6 @@ jobs:
           organization: EddieHubCommunity
           label: invite me to the organisation
           repo-token: ${{ secrets.GITHUB_TOKEN }}
-          comment: '<b>Invitation sent for the GitHub Organisation. Welcome to the community 🎉</b><br><br>Don''t forget after accepting to make it public so it appears on your GitHub profile for everyone else to see, you can do this by finding your name in the GitHub organisation list and change the dropdown to public https://github.com/orgs/EddieJaoudeCommunity/people<br><br>Tips for practising:<br><br>Customise your GitHub profile, here is a live stream on it https://www.youtube.com/watch?v=cT6GXCuS0Zo<br>Practice repo, instructions on how to add your name to the README in the README https://github.com/EddieJaoudeCommunity/hacktoberfest-practice<br>Remember contributing to open source is not just about code, its about collaboration, communication and adding value<br>I hope that helps'
+          comment: '<b>Invitation sent for the GitHub Organisation. Welcome to the community 🎉</b><br><br>Don''t forget after accepting to make it public so it appears on your GitHub profile for everyone else to see, you can do this by finding your name in the GitHub organisation list and change the dropdown to public https://github.com/orgs/EddieHubCommunity/people<br><br>Tips for practising:<br><br>Customise your GitHub profile, here is a live stream on it https://www.youtube.com/watch?v=cT6GXCuS0Zo<br>Practice repo, instructions on how to add your name to the README in the README https://github.com/EddieJaoudeCommunity/hacktoberfest-practice<br>Remember contributing to open source is not just about code, its about collaboration, communication and adding value<br>I hope that helps'
```

```suggestion
          comment: '<b>Invitation sent for the GitHub Organisation. Welcome to the community 🎉</b><br><br>Don''t forget after accepting to make it public so it appears on your GitHub profile for everyone else to see, you can do this by finding your name in the GitHub organisation list and change the dropdown to public https://github.com/orgs/EddieHubCommunity/people<br><br>Tips for practising:<br><br>Customise your GitHub profile, here is a live stream on it https://www.youtube.com/watch?v=cT6GXCuS0Zo<br>Practice repo, instructions on how to add your name to the README in the README https://github.com/EddieHubCommunity/hacktoberfest-practice<br>Remember contributing to open source is not just about code, its about collaboration, communication and adding value<br>I hope that helps'
```
Can the link to the hacktoberfest-practice repo also be updated?

---

As always you spot everything 👀